.github/workflows/hqs-build-deploy-rust-pyo3.yml

```diff
@@ -6,60 +6,63 @@ on:
     types: [created]
 
 jobs:
-
   deploy_cratesio:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4.1.6
-    # - uses: Swatinem/rust-cache@v2
-    - uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: stable
-        default: true
-    - name: setup cargo
-      run: |
-        # Authentication at crates.io with HQS organizational token
-        cargo login ${{ secrets.CRATES_IO_API_TOKEN }}
-        # publishing a specific package to crates.io
-        cargo publish --manifest-path=roqollage/Cargo.toml
-        # Wait to publish next package (e.g. the Python interface)
-        # that depends on previous package so that crates.io is
-        # updated and accepts the upload
-        sleep 60
-        # Publish second package
-        cargo publish --manifest-path=qollage/Cargo.toml
+      - uses: actions/checkout@v4.1.6
+      # - uses: Swatinem/rust-cache@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          default: true
+      - name: setup cargo
+        run: |
+          # Authentication at crates.io with HQS organizational token
+          cargo login ${{ secrets.CRATES_IO_API_TOKEN }}
+          # publishing a specific package to crates.io
+          cargo publish --manifest-path=roqollage/Cargo.toml
+          # Wait to publish next package (e.g. the Python interface)
+          # that depends on previous package so that crates.io is
+          # updated and accepts the upload
+          sleep 60
+          # Publish second package
+          cargo publish --manifest-path=qollage/Cargo.toml
 
-# Build linux wheels and upload them to PyPi
+  # Build linux wheels and upload them to PyPi
   build_maturin_builds_linux:
     name: maturin_build-linux
     needs: deploy_cratesio
     uses: HQSquantumsimulations/reusable_workflows/.github/workflows/reusable_build_maturin_linux.yml@main
-    with: 
+    with:
       py_interface_folder: "qollage"
       deploy: true
       # Whether to build for aarch64. Can fail if there are C/C++ dependencies
-      build_for_arm: true
+      build_for_arm: false
```

Comment:
How come this was changed?

Reply:
`ring` a dependency of `typst` doesn't support building for arm. Right now the arm building job is failing during releases so I took it out